projects/nx-verdaccio/src/plugin/caching.ts

```diff
@@ -45,7 +41,7 @@ export function readTargetsCache(
 export function writeTargetsToCache(
   cachePath: string,
   results: Record<string, Partial<ProjectConfiguration>>
-) {
+): void {
   process.env.NX_CACHE_PROJECT_GRAPH !== 'false' &&
     writeJsonFile(cachePath, results);
```

Comment:
We could use writeJsonFileAsync instead of writeJsonFile to make the code non blocking.

Reply:
@BioPhoton What do you think about doing it in seperate pr?